docs/examples/driver_examples/calibrate_keithley.py

```diff
@@ -0,0 +1,215 @@
+#%% Function definitions
+import time
+from qcodes.instrument.base import Instrument
+from qcodes.instrument.parameter import Parameter
+
+
+src_FS_map = {"200e-3": 180e-3, "2": 1.8, "20": 18, "200": 180}
+channels = ["smua", "smub"]
+
+
+def save_calibration(smu: Instrument) -> None:
+    for channel in channels:
+        smu.write(f"{channel}.cal.save()")
+
+
+def calibrate_keithley_smu_v(
+    smu: Instrument,
+    dmm_param: Parameter,
+    src_Z: float = 1e-30,
+    time_delay: float = 3.0,
+    save_calibrations: bool = False,
+) -> None:
+    smu_ranges = ["200e-3", "2", "20"]
+    dmm_ranges = [1, 10, 100]
+    for channel in channels:
+        input(f"Please connect channel {channel} to V input on calibrated DMM.")
+        for smu_range, dmm_range in zip(smu_ranges, dmm_ranges):
+            dmm_param.range(dmm_range)
```

Comment:
what is dmm_param? isn't it the dmm instrument actually? or does dmm have a parameter that has a fancy custom range method on it?

Reply:
Fixed, it was supposed to be the DMM instrument.